### PR TITLE
WBJetsToLNu_Wpt-80toInf_5f_LO_MLM DYBJetsToNuNu_Zpt-80toInf_M-50_5f_L…

### PR DESCRIPTION
**Pull request by perrozzi**
same as https://github.com/cms-sw/genproductions/pull/591 with 
80  = ptllmin  ! Minimum pt for 4-momenta sum of leptons(l and vl)

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+define j = p
+define bquark = b b~
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+# Specify process(es) to run
+generate p p > ell+ vl bquark / h @0
+add process p p > ell+ vl bquark j / h @1
+add process p p > ell+ vl bquark j j j / h @2
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM_proc_card.dat]: these are "out of order" which might cause confusion downstream when looking at the matching efficiency tables

**Comment by bendavid** [comment]
Could do the same with the decay syntax + bwcutoff for the znunu here in principle.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM_proc_card.dat (hunk 1 above)
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM_proc_card.dat]: fixed, thanks

**Comment by perrozzi** [comment]
done

**Comment by bendavid** [comment]
Ah, so in fact the decay chain syntax plus large bwcutoff should ALSO be used for the w samples.
(This is how we did the inclusive samples in this round also)
Otherwise the 4 jet case with off-diagonal ckm becomes quite intensive.

**Comment by perrozzi** [comment]
no problem, done

**Pull request merged**